client/post-editor/media-modal/index.jsx

```diff
@@ -145,16 +146,18 @@ export class EditorMediaModal extends Component {
 			return;
 		}
 
-		const value = mediaLibrarySelectedItems.length
+		if ( mediaLibrarySelectedItems && this.state.source !== '' ) {
```

Comment:
As far as I know `mediaLibrarySelectedItems` is always an array, so this check will always pass. Previously it was checking the length of the array.

I'm unsure how `confirmSelection` can be called without any items, so I'm not even sure the check is necessary?